### PR TITLE
removing beacon functionality from the start_accepting function.

### PR DESCRIPTION
**Pull request by inetic**
[Review on Reviewable](https://reviewable.io/reviews/maidsafe/crust/289)

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
r? @vinipsmaker

(maidsafe_highfive has picked a reviewer for you, use r? to override)

**Comment by inetic** [comment]
fixes #287

File: src/transport.rs [hunk 1]
```diff
@@ -38,6 +39,14 @@ pub enum Endpoint {
 }
 
 impl Endpoint {
+    /// Construct a new Endpoint
+    pub fn new(addr: IpAddr, port: Port) -> Endpoint {
```
Comment (Fraser999) [inline src/transport.rs]: Nice one!  Cleans up a fair bit of code.

**Pull request merged**